src/helpers/certs-helpers.ts

```diff
@@ -1,9 +1,10 @@
+import { X509Certificate } from 'node:crypto';
 import { join } from 'node:path';
 import { ensureDirSync, readFileSync, readdirSync } from 'fs-extra';
 import { userDataCertsPath } from '../environment-remote';
 import { removeNewLines } from '../jsUtils';
 
-export const checkIfCertIsPresent = (certData: string): boolean => {
+export const checkIfCertIsPresent = (clientCert): boolean => {
```

Comment:
Can you please add the type of the parameter? It should be of type `Certificate` from `electron`.